src/models/questionCard.model.js

```diff
@@ -0,0 +1,63 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const questionCardSchema = new Schema({
+    paragraphs: {
+        type: String,
+    },
+    question: [
+        {
+            hint: {
+                type: String,
+            },
+            image: {
+                type: String,
+                default: null,
+            },
+            sound: {
+                type: String,
+                default: null,
+            },
+            texts: {
+                type: String,
+                required: true,
+                trim: true,
+            },
+        },
+    ],
+    answer: [
+        {
+            correct: {
+                type: String,
+                required: true,
+            },
+            choices: [
+                {
+                    type: String,
+                    required: true,
+                },
+            ],
+        },
+    ],
```

Comment:
đoạn này bạn sửa như form này nhé
```suggestion
children: [
    {
      question: {}
      answer: {}
    }
]
```